index.html

```diff
@@ -217,10 +217,34 @@ <h3 class="Frameworks">Skills</h3>
                     </li>
                 </ul>
             </div>
-
-
         </section>
     </section>
+    <!-- *****Contact me****** -->
+    <section class="footer">
```

Comment:
Excellent job adding the right path 👍 Your path is fixed now, but you still need to take a small step to reach the finish line 
- [ ] Please make sure that you follow the figma design as much as possible according to the [requiremnts](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Following is a quote for a related requirement 🔽 Please fix this by adding the correct background color to contact section. I know you can do that 💪

  > _You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)._

## Your design previously 👇
![image](https://user-images.githubusercontent.com/106119105/213175090-6482acb9-0de2-4916-b527-256189ce9324.png)

## Your design now 👇
![image](https://user-images.githubusercontent.com/106119105/213175367-95876933-79f5-48a5-9469-21cd606add32.png)


## Figma design 👇
![image](https://user-images.githubusercontent.com/106119105/213175520-69890d23-b759-4aac-ab4f-e43c43425da0.png)